src/pivot-table/hooks/use-column-width.ts

```diff
@@ -94,14 +99,14 @@ export default function useColumnWidth(
         // Use the max width of all measures
         width = Math.max(
           ...qMeasureInfo.map(({ qFallbackTitle, columnWidth }) => {
-            const fitToContentWidth = measureTextForContent(qFallbackTitle);
+            const fitToContentWidth = measureTextForContent(qFallbackTitle) + TOTAL_CELL_PADDING;
```

Comment:
The text would not fit if the cell padding was not added

Reply:
Is it only the total label that doesn't fit or is it a general problem?

Reply:
General problem since width of the cell is equal to the width of the text. But the cell has padding so the text it would not fit.